lib/msal-common/src/utils/TimeUtils.ts

```diff
@@ -15,6 +15,31 @@ export function nowSeconds(): number {
     return Math.round(new Date().getTime() / 1000.0);
 }
 
+/**
+ * Returns current time in JS Date object with offset
+ * @param offset
+ */
+export function nowDateWithOffset(offset: number): Date {
```

Comment:
Can we specify units in the var name and doc block above?

Reply:
You mean like `nowDateMillisecondsWithOffset`?

Reply:
Sorry, the param name: `offset` -> `offsetMs`